PresentScreenings/ListReader.cs

```diff
@@ -8,39 +8,40 @@ namespace PresentScreenings.TableView
     /// List reader, read a file and return is list of generic objects.
     /// </summary>
 
-    public class ListReader<T> where T : class
-	{
-		#region Private Members
-		string _fileName;
-		bool _skipHeader;
-		#endregion
-
-		#region Constructors
-		public ListReader(string fileName, bool skipHeader = false)
-		{
-			_fileName = fileName;
-			_skipHeader = skipHeader;
-		}
-		#endregion
+    public abstract class ListReader<T> where T : class, new()
+    {
+        #region Virtual Methods
+        public virtual bool ListFileIsMandatory()
+        {
+            return true;
+        }
+        public virtual bool ListFileHasHeader()
+        {
+            return true;
+        }
+        #endregion
 
-		#region Public Methods
-		public List<T> ReadListFromFile(Func<string, T> TConstructor)
-		{
-			var resultList = new List<T> { };
+        #region Public Methods
+        public List<T> ReadListFromFile(string _fileName, Func<string, T> lineConstructor)
```

Comment:
Please rename the parameter name `_fileName` as to not look like a private member.

Reply:
Done.
Thanks for finding this!